src/main/fc/cli.c

```diff
@@ -2211,8 +2211,8 @@ static void cliDshotProg(char *cmdline)
                 int command = atoi(pch);
                 if (command >= 0 && command < DSHOT_MIN_THROTTLE) {
                     if (escNumber == ALL_ESCS) {
-                        for (unsigned i = 0; i < getMotorCount(); i++) {
-                            pwmWriteDshotCommand(i, command);
+                        for (uint8_t index = 0; index < getMotorCount(); index++) {
```

Comment:
Thinking about it some more, I'm surprised the compiler doesn't pick up on that optimization.   I got it changed, but more curious about it.